src/FreeDSx/Snmp/Protocol/SnmpEncoder.php

```diff
@@ -46,10 +46,12 @@ class SnmpEncoder extends BerEncoder
 {
     public function __construct(array $options = [])
     {
-        parent::__construct([
-            'primitive_only' => [
-                AbstractType::TAG_TYPE_OCTET_STRING,
-            ],
-        ]);
+        parent::__construct(
+            $options + [
+                'primitive_only' => [
+                    AbstractType::TAG_TYPE_OCTET_STRING,
+                ],
+            ]
+        );
```

Comment:
I'd actually just remove the options from the constructor altogether from this class. It should not be added. Probably got added automagically by PhpStorm when I created the class and I never removed it.